app/components/PerusePages/History/History.tsx

```diff
@@ -114,6 +119,9 @@ export class History extends Component<HistoryProps, {}> {
                             <H1 title="History" />
                         </PageHeader>
                     </Page>
+                    <h6 className={`${styles.warning}`}>
+            *Note: Time displayed is in GMT format
```

Comment:
I dont think we need this warning TBH.

Reply:
We add the warning to notify users about the timeStamps being in GMT format, while they would usually expect a local timeStamp.

Reply:
So, I think GMT is a timezone and not a format. You're looking for 24hr of 12hr format. (12hr being w/ AM/PM).

But I also don't think we need a note for this.